geojson/validation.py

```diff
@@ -58,14 +58,19 @@ def is_valid(obj):
                           'must be equivalent')
 
     if isinstance(obj, geojson.MultiPolygon):
-        if checkListOfObjects(obj['coordinates'],
-                              lambda x: len(x) >= 4 and x[0] == x[-1]):
+        if checkListOfObjects(obj['coordinates'], lambda x: is_polygon(x)):
```

Comment:
Can you wrap this line? It's one character too long for flake8 :P 

https://travis-ci.org/frewsxcv/python-geojson/jobs/84943678